.github/workflows/test_iree.yml

```diff
@@ -65,7 +65,30 @@ jobs:
       - name: "Running the generated ONNX test suite"
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest iree_tests/onnx/node/generated -n auto -rpfE --timeout=30 --retries 2 --retry-delay 5 --durations=10
+          pytest iree_tests/onnx/node/generated \
+            -n auto \
+            -rpfE \
+            --timeout=30 \
+            --retries=2 \
+            --retry-delay=5 \
+            --durations=10 \
+            --report-log=/tmp/iree_tests_onnx_cpu_logs.json \
+            --config-files=./iree_tests/configs/config_onnx_cpu_llvm_sync.json
+
+      - name: "Updating config file with latest XFAIL lists"
+        if: failure()
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python iree_tests/update_config_xfails.py \
+            --log-file=/tmp/iree_tests_onnx_cpu_logs.json \
+            --config-file=./iree_tests/configs/config_onnx_cpu_llvm_sync.json
+          cat ./iree_tests/configs/config_onnx_cpu_llvm_sync.json
+      - name: "Uploading new config file"
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: "config_onnx_cpu_llvm_sync.json"
+          path: "./iree_tests/configs/config_onnx_cpu_llvm_sync.json"
```

Comment:
Yay more copy/paste boilerplate... more reasons to use job matrices :P 
https://github.com/iree-org/iree/issues/17136